jib-core/src/test/java/com/google/cloud/tools/jib/tar/TarExtractorTest.java

```diff
@@ -92,16 +94,20 @@ public void testExtract_modificationTimePreserved() throws URISyntaxException, I
 
     TarExtractor.extract(source, destination);
 
+    // We have to use millisecond precision here. Taken from Instant.parse.
+    final DateTimeFormatter parser =
+        new DateTimeFormatterBuilder().parseCaseInsensitive().appendInstant(3).toFormatter();
+
```

Comment:
@izogfif Is there is a reason we're testing with millisecond precision here? Curious since we opted for nanosecond precision above (ReproducibleLayerBuilder)

Reply:
For some reason, the code below can't extract micro- and nanoseconds, it's limited to milliseconds (at least on Windows machine with NTFS).
In one of the methods below I had to add centiseconds and avoided having to use the same DateTimeFormatter since, conveniently, truncation to milliseconds yielded "0" as the third digit, and Instant.parse can parse centiseconds.